Formula/gcc-arm-embedded.rb

```diff
@@ -0,0 +1,64 @@
+class GccArmEmbedded < Formula
+  desc "GNU Embedded Toolchain for Arm Cortex-M/R processors"
+  homepage "https://developer.arm.com/open-source/gnu-toolchain/gnu-rm"
+  url "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/gcc-arm-none-eabi-9-2019-q4-major-src.tar.bz2"
+  version "9-2019-q4-major"
+  sha256 "f162a655f222319f75862d7aba9ff8a4a86f752392e4f3c5d9ef2ee8bc13be58"
+
+  depends_on "gnu-tar" => :build
+  depends_on :xcode => ["10.0", :build]
+  uses_from_macos "expat"
+  depends_on "gmp"
+  depends_on "isl"
+  depends_on "libelf"
+  depends_on "libmpc"
+  depends_on "mpfr"
+
+  def install
+    ENV.deparallelize
```

Comment:
All the components (binutils, gcc) should build fine in parallel. What is the problem?

Reply:
The build fails when it is run in parallel like this. There are specific makefiles that can be run in parallel and the build will succeed on Catalina, but not on High Sierra and Mojave if they use APFS. This bug seems to be related to an APFS issue that looks like it is fixed on Catalina.

I referenced this on a response to a prior change you requested (https://github.com/Homebrew/homebrew-core/pull/46584#discussion_r344475336).

Reply:
I'm sorry, we have some many PRs open, it's sometimes hard to keep track. Right now, a 4-hour build for a formula with minor usage is not something we can have in Homebrew core, I'm sorry.

Regarding parallel building, if you go to the GCC bugzilla issue, you'll see it was reported by me, actually… 😄  and it was fixed, so it should definitely not be an issue anymore. We build many GCC versions in parallel, in the `gcc*` formulas.

Reply:
I didn't even realize that lol! What're the odds. And I understand, I do think this formula would get some usage due to the gcc arm embedded compiler being needed for compiling [CircuitPython](https://github.com/adafruit/circuitpython), but I see your point. I am hoping I can get this formula to a place that is acceptable as it would be my first formula contribution, which is somewhat exciting.

I will change the formula and put back the `inreplace` for parallel building. However, I cannot test this on my own machine for the High Sierra and Mojave builds. So I will just have to see what happens with the test bot. Is this okay?